mysql-test/suite/rocksdb/t/ttl_primary_with_partitions.test

```diff
@@ -0,0 +1,245 @@
+--source include/have_rocksdb.inc
```

Comment:
Still --source include/have_partition.inc unless, as with the previous test, there is something in the next merge?